Shared/Localization/en.lproj/Localizable.strings

```diff
@@ -507,6 +508,7 @@
 "CACHE_SIZE_%@" = "Cache size: %@";
 "SUPPORT_VIA_KOFI" = "Support via Ko-Fi";
 "UNREAD_CHAPTERS" = "Unread Chapters";
+"AS_ISOLATE_PAGE" = "As Isolate Page";
```

Comment:
i think a more natural sounding name for this would be something like this. isolate is technically the right word but a little awkward used in this scenario
```suggestion
"SET_AS_SINGLE_PAGE" = "Set as Single Page";
```